bigtable/google/cloud/bigtable/client.py

```diff
@@ -76,8 +76,15 @@ def _make_data_stub(client):
     :returns: A gRPC stub object.
     """
     if client.emulator_host is None:
+        one_hundred_mb = 100 * 1024 * 1024
+        # NOTE: 'grpc.max_message_length' will be deprecated in the 1.1 release
+        # of grpcio in favor of 'grpc.max_receive_message_length' and
+        # 'grpc.max_send_message_length'.
+        max_msg_length = (('grpc.max_message_length', one_hundred_mb),
```

Comment:
Having no relationship to the enclosing function's parameters I'd offer that this value should also be a module-scope module-private constant.